spec/reports/oclc_registration_spec.rb

```diff
@@ -28,7 +33,7 @@
       "FOO",
       "123",
       "committed to retain",
-      com.committed_date,
+      com.committed_date.strftime("%Y%m%d"),
```

Comment:
Duplicates logic in the code, but seems hard to avoid unless we set `committed_date` to a known value.